webook/arrangement/views/person_views.py

```diff
@@ -270,3 +269,23 @@ def get(self, request, *args, **kwargs):
 
 
 people_calendar_resources_list_view = PeopleCalendarResourcesListView.as_view()
+
+
+class PersonSearchPlannersView(SearchPeopleAjax):
+    def search(self, search_term):
+        show_only_planners: bool = self.request.GET.get("only_planners", False)
```

Comment:
We should remove this logic - - not used in front-end, and I left it in with future-proofing in mind. But it will likely never be necessary. Easy to re-implement anywho.